runtime/vm/jvminit.c

```diff
@@ -588,6 +588,12 @@ cleanCustomSpinOptions(void *element, void *userData)
 }
 #endif /* J9VM_INTERP_CUSTOM_SPIN_OPTIONS */
 
+BOOLEAN
```

Comment:
Why have this as a function instead of a macro? Is there someone that really needs to call it indirectly?

Reply:
The JIT will need it